src/WebJobs.Script.ConsoleHost/Verbs/ListVerb.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using NCli;
+using WebJobs.Script.ConsoleHost.Arm;
+
+namespace WebJobs.Script.ConsoleHost.Verbs
+{
+    [Verb(HelpText = "Lists function apps in current tenant. See `switch-tenant` command")]
```

Comment:
nit: make casing consistent across all output. You use "Function App" other places.